src/integration/test_suites/libs/sha256/sha256.c

```diff
@@ -64,7 +64,7 @@ void sha256_flow(sha256_io block, uint8_t mode, uint8_t wntz_mode, uint8_t wntz_
     
     // wait for SHA to be valid
     wait_for_sha256_intr();
-
+    cptra_intr_rcv.sha256_notif &= ~SHA256_REG_INTR_BLOCK_RF_NOTIF_INTERNAL_INTR_R_NOTIF_CMD_DONE_STS_MASK;
```

Comment:
IMO, clearing the variable should be done inside `wait_for_sha256_intr` to make sure it's applied uniformly. (e.g.  `wait_for_sha256_intr` is used below in a different context, and the variable isn't cleared there).

Reply:
Oh good point. I missed that. I'll move it to inside the wait function. Thanks!